e2e/testcases/sync_ordering_test.go

```diff
@@ -35,7 +35,7 @@ import (
 // This file includes e2e tests for sync ordering.
 // The sync ordering feature is only supported in the multi-repo mode.
 
-func TestSyncOrdering(t *testing.T) {
+func TestSyncOrdering_CM_HappyPath(t *testing.T) {
```

Comment:
Not a fan of HappyPath as a name. Can you make it more meaningful? Like TestMultipleDependencies?

Also not sure why you added underscores. Most of the other test names just use camel case, right?

Reply:
We do have a dozen of tests using underscores like `TestConflictingDefinitions_NamespaceToNamespace`. It is not consistent. 
I renamed it to TestMultipleDependencies.